src/cli.rs

```diff
@@ -159,6 +159,13 @@ Defaults to \"format\" if the NO_COLOR env is set and to \"none\" if stdout is n
     #[clap(short = 'd', long)]
     pub download: bool,
 
+    /// During download, keep the raw encoding of the body. Intended for use with download or
+    /// to debug an encoded response body.
+    ///
+    /// For example, set Accept-Encoding: gzip and use --preserve-encoding to skip decompression.
+    #[clap(long)]
+    pub preserve_encoding: bool,
+
```

Comment:
Let's shorten the first line to fit within ~80 chars. Any extended help can go in subsequent lines.

(to see the difference, try `cargo run -- --help` and `cargo run -- help`)

```rs
/// Skip decompressing the response body
///
/// longer help
```

Also, should we move this option to after `response_mime` to improve discoverability?